starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaDrmBridge.java

```diff
@@ -373,6 +373,11 @@ private MediaDrmBridge(String keySystem, UUID schemeUUID, long nativeMediaDrmBri
     mSchemeUUID = schemeUUID;
     mMediaDrm = new MediaDrm(schemeUUID);
 
+    // Get info of hdcp connection
+    if (Build.VERSION.SDK_INT >= 28) {
```

Comment:
This has to be 29, as HDCP_V2_3 is only available in 29.

Reply:
Fixed.